.travis.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       deploy:
           on:
             branch: master
-            tags: true
+            condition: $TRAVIS_COMMIT_MESSAGE =~ ^chore\(release\):\ ([0-9]).([0-9]).([0-9])|(-[0-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)$
```

Comment:
@PixnBits does this look okay to you ?

Reply:
i think this will break when we get to `10.0.0` or `1.10.0` or `1.0.10` also the pre-release group needs to account for `.` 

can you validate this works `chore(release): 10.11.20-build.alpha.016` and `chore(release): 109.111.220-build-016`

Reply:
I dont think any validation after `chore(release):` is needed

Reply:
working on the regex for this took me to the pits of regex - i used the http://semver.org/ regex